corehq/apps/export/filters.py

```diff
@@ -0,0 +1,114 @@
+from corehq.apps.es import filters as esfilters
```

Comment:
If we also run exports off of couch, I was imagining that each of these classes would have a second method like `to_couch_filter()` or something like that, which would return a function that could be used to filter couch documents.

If we decide to do everything from ES, then we probably don't need these intermediary classes. Although if we had the django views create instances of these generic filters, it might create a nice barrier between the model and presentation.